cli/config/config.go

```diff
@@ -228,13 +232,31 @@ func initConfig() {
 
 func DisplayUsedConfig() {
 	// print config file
-	if !LoadedConfig && len(UserProvidedPath) > 0 {
-		log.Warn().Msg("could not load configuration file " + UserProvidedPath)
-	} else if LoadedConfig {
+	if LoadedConfig {
 		log.Info().Msg("loaded configuration from " + viper.ConfigFileUsed() + " using source " + Source)
 	} else if Source == configSourceBase64 {
 		log.Info().Msg("loaded configuration from environment using source " + Source)
 	} else {
 		log.Info().Msg("no Mondoo configuration file provided. using defaults")
 	}
 }
+
+func ValidateUserProvidedConfigPath() error {
+	configPath := os.Getenv(configSourcePath)
+	if !LoadedConfig && len(UserProvidedPath) > 0 {
+		_, err := os.Stat(UserProvidedPath)
+		if err != nil {
+			return errors.New(fmt.Sprintf("Could not load config: %v - file does not exist", UserProvidedPath))
```

Comment:
I think we want to introduce specific error types and then attach the text to the errors. This makes it easier to check validation programmatically and we can make the log.Fatal output more nice since we know it is an expected error. https://go.dev/blog/error-handling-and-go is a good reading for that.